google/cloud/security/common/gcp_api/cloud_resource_manager.py

```diff
@@ -114,16 +114,17 @@ def get_project_iam_policies(self, resource_name, project_numbers):
         """
         projects_stub = self.service.projects()
 
-        try:
-            for project_number in project_numbers:
+        for project_number in project_numbers:
```

Comment:
I think this works fine for now. My comment on accepting either project_number or project_id was more to encourage code reuse if some other bit of code wanted to call get_project_iam_policies and only had the project_id, for example. Like if a future enforcer update wants to check the current policy before enforcing changes on it.